### PR TITLE
Add extended exposure settings to improve low light performance.

### PR DESCRIPTION
**Pull request by sveip**
## Description:
Doc for PR https://github.com/esphome/esphome/pull/2892

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#2892

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/esp32_camera.rst [hunk 1]
```diff
@@ -92,6 +92,9 @@ Frame Settings:
 - **saturation** (*Optional*, int): The saturation to apply to the picture, from -2 to 2. Defaults to ``0``.
 - **vertical_flip** (*Optional*, boolean): Whether to flip the image vertically. Defaults to ``true``.
 - **horizontal_mirror** (*Optional*, boolean): Whether to mirror the image horizontally. Defaults to ``true``.
+- **aec2** (*Optional*, bool): Whether to enable Auto Exposure Control 2. Defaults to ``false``.
```
Comment (oxan) [inline components/esp32_camera.rst]:
```suggestion
- **aec2** (*Optional*, boolean): Whether to enable Auto Exposure Control 2. Defaults to ``false``.
```

File: components/esp32_camera.rst [hunk 2]
```diff
@@ -92,6 +92,9 @@ Frame Settings:
...
+- **ae_level** (*Optional*, int): The auto exposure leve to apply to the picture, from -2 to 2. Defaults to ``0``.
```
Comment (oxan) [inline components/esp32_camera.rst]:
```suggestion
- **ae_level** (*Optional*, int): The auto exposure level to apply to the picture, from -2 to 2. Defaults to ``0``.
```

**Review by oxan — CHANGES_REQUESTED** [review]
This PR should be rebased on `next`.

**Comment by sveip** [comment]
> * Fixes, changes and adjustments for the current release should be created against `current`.

Ok, I read this and chose current :)
`Fixes, changes and adjustments for the current release should be created against current.`
I'm not infront of a computer atm, if you can, please rebase against `next`.

Edit: I was able to change it myself from the webui on my phone :-)

**Comment by oxan** [comment]
> Ok, I read this and chose current :)
> `Fixes, changes and adjustments for the current release should be created against current.`

This is documentation for new config options introduced by a PR that isn't part of  the current release, so this doesn't apply.

> I'm not infront of a computer atm, if you can, please rebase against `next`.
> 
> Edit: I was able to change it myself from the webui on my phone :-)

Just changing the target branch of the PR isn't enough. The branch of the PR should be rebased onto the next branch, as it now tries to merge commits from the `current` branch into the `next` branch.

**Comment by sveip** [comment]
@oxan, I've commited it to the `next` branch on my fork, but I can't find any option to change it here in the PR. Should I close this and make a new one?

**Comment by oxan** [comment]
Creating a new PR is fine.

**Pull request closed without merging**